FrontEnd/src/components/basics/ImageUpload/ImageUpload.module.scss

```diff
@@ -0,0 +1,71 @@
+$border-radius: 0.1875rem;
+.image_input {
```

Comment:
Shouldn't use underline (snake case), should use camelcase to stay consistent with the rest of the codebase (goes for the rest of the scss too)

Reply:
loool funny